projects/ddap-common-lib/src/lib/layout/menu/menu.component.html

```diff
@@ -4,13 +4,26 @@
 -->
 <mat-toolbar color="primary" class="main-header">
   <div class="menus">
-    <img *ngIf="logoUrl" class="m-r" style="height: 32px; width: auto;" src="{{logoUrl}}"/>
-    <span *ngIf="isSandbox"
-      class="sandbox-warning"
-      matTooltip="This is sandbox testing environment">
-      Sandbox environment
-    </span>
-    <a class="site-name" routerLink="/">{{siteName}}</a>
+    <ng-container *ngIf="logoLink; else noLogoLink">
+      <a href="{{logoLink}}">
+        <img *ngIf="logoUrl" class="m-r" style="height: 32px; width: auto;" src="{{logoUrl}}"/>
```

Comment:
@monicavalluri @mpanik What do you think if we have the logo and the site name in the same anchor tag?